hibernate-core/src/main/java/org/hibernate/id/factory/IdentifierGeneratorFactory.java

```diff
@@ -71,5 +71,5 @@ Generator createIdentifierGenerator(
 	 * @deprecated use {@link #createIdentifierGenerator(GenerationType, String, String, JavaType, Properties, GeneratorDefinitionResolver)}
 	 */
 	@Deprecated(since = "6.0")
-	Generator createIdentifierGenerator(String strategy, Type type, Properties parameters);
+	Generator createIdentifierGenerator(String strategy, Type type, GeneratorCreationContext creationContext, Properties parameters);
```

Comment:
This is a change in an API, though namely a deprecated one, but users might be impacted.

Reply:
I think it's quite unlikely that any user programs were _ever_ relying on this SPI. And yes, it's been deprecated since H6.

(Though I guess it's possible Envers or Search might.)

But if you guys think there's a risk here, it's easy enough to rescue the previous signature.

Reply:
I have restored the old signature.